ocs_ci/ocs/ui/deployment_ui.py

```diff
@@ -279,6 +279,26 @@ def verify_operator_succeeded(
             )
             raise TimeoutExpiredError
 
+    def verify_noobaa_ui(self):
+        """
+        Verify Noobaa UI
+
+        """
+        self.navigate_overview_page()
+        self.do_click(self.dep_loc["object_service_tab"])
+        platform = config.ENV_DATA.get("platform").lower()
+        if platform == constants.VSPHERE_PLATFORM:
```

Comment:
any plans to add verification for AWS ? or we have it covered somewhere else

Reply:
On AWS, I  search "strings" on relevant html